redfish-core/lib/ethernet.hpp

```diff
@@ -1660,11 +1831,141 @@ inline void handleIPv6StaticAddressesPatch(
+        staticRouteEntry = staticRouteData.cbegin();
+    for (const nlohmann::json& thisJson : input)
+    {
+        std::string pathString = "StaticRoutes/" + std::to_string(entryIdx);
+
+        if (!thisJson.is_null() && !thisJson.empty())
+        {
+            std::optional<std::string> destination;
+            std::optional<std::string> gateway;
+            std::optional<uint32_t> prefixLength;
+            nlohmann::json thisJsonCopy = thisJson;
+            if (!json_util::readJson(thisJsonCopy, asyncResp->res,
+                                     "Destination", destination, "Gateway",
+                                     gateway, "PrefixLength", prefixLength))
+            {
+                messages::propertyValueFormatError(
+                    asyncResp->res,
+                    thisJson.dump(2, ' ', true,
+                                  nlohmann::json::error_handler_t::replace),
+                    pathString);
+                return;
+            }
+
+            const std::string* gw = nullptr;
+            const std::string* dest = nullptr;
+            uint32_t prefix = 0;
+
+            if (destination)
+            {
+                dest = &(*destination);
+            }
+            else if (staticRouteEntry != staticRouteData.end())
+            {
+                dest = &(staticRouteEntry->destination);
+            }
+            else
+            {
+                messages::propertyMissing(asyncResp->res,
+                                          pathString + "/Destination");
+                return;
+            }
+
+            if (gateway)
+            {
+                gw = &(*gateway);
+            }
+            else if (staticRouteEntry != staticRouteData.end())
+            {
+                gw = &(staticRouteEntry->gateway);
+            }
+            else
+            {
+                messages::propertyMissing(asyncResp->res,
+                                          pathString + "/Gateway");
+                return;
+            }
+
+            if (prefixLength)
+            {
+                prefix = *prefixLength;
+            }
+            else if (staticRouteEntry != staticRouteData.end())
+            {
+                prefix = staticRouteEntry->prefixLength;
+            }
+            else
+            {
+                messages::propertyMissing(asyncResp->res,
+                                          pathString + "/PrefixLength");
+                return;
+            }
+
+            if (staticRouteEntry != staticRouteData.end())
+            {
+                deleteAndCreateStaticRoute(ifaceId, staticRouteEntry->id, *dest,
+                                           *gw, prefix, asyncResp);
+                staticRouteEntry++;
+            }
+            else
+            {
+                createStaticRoute(ifaceId, *dest, *gw, prefix, asyncResp);
+            }
+            entryIdx++;
+        }
+        else
```

Comment:
Don't you also end up down here if thisJson.empty() ? What happens in that case ?
Can you add a comment or 2, it would help reviewers follow this code

Reply:
its  no-op if json.emtpy(), I see the same followed in other places

Reply:
Can you add a comment or 2 in this code?

Reply:
its followed everywhere, I don't think we need explicit comment here..